src/style.scss

```diff
@@ -0,0 +1,144 @@
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.container {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid white;
+  background-color: white;
+  text-align: center;
+  overflow: hidden;
+
+  [data-qa="card"] {
+    padding: 16px;
+
+    .card__image {
+      width: 160px;
+      height: 134px;
+      margin-top: 32px;
+      margin-bottom: 0;
+      overflow: hidden;
+
+      img {
+        width: 100%;
+        height: 100%;
+        object-fit: contain;
+      }
+    }
+
+    .card__title {
+      margin: 16px 0 0 ;
+      margin-top: 40px;
+      font-family: Roboto, sans-serif;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      text-align: left;
+      color: #060b35;
+    }
+
+    .product-code {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      text-align: left;
+      color: #616070;
+      margin-top: 4px;
+    }
+
+    .reviews {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      margin-top: 10px;
+
+      .reviews__stars {
```

Comment:
Consider using styles from the previous task

Reply:
Not fixed, according to the task description
![image](https://github.com/mate-academy/layout_product-cards/assets/80700247/6363bfd5-37c1-439a-b2f6-6bb6c492e42b)